docs/troubleshooting/troubleshoot_deployment.md

```diff
@@ -80,19 +92,48 @@ variable for your `data-persistence` module to increase the number of instances:
 }
 ```
 
-## Install dashboard
+## Install Dashboard
 
-### Dashboard configuration
+### Dashboard Configuration
 
-Issues:
+#### Issues
 
-- **Problem clearing the cache: EACCES: permission denied, rmdir '/tmp/gulp-cache/default'**", this probably means the files at that location, and/or the folder, are owned by someone else (or some other factor prevents you from writing there).
+::::note Not Able To Clear Cache
+
+**Problem clearing the cache: EACCES: permission denied, rmdir '/tmp/gulp-cache/default'**", this probably means the files at that location, and/or the folder, are owned by someone else (or some other factor prevents you from writing there).
+
+:::tip Workaround Option
 
 It's possible to workaround this by editing the file `cumulus-dashboard/node_modules/gulp-cache/index.js` and alter the value of the line `var fileCache = new Cache({cacheDirName: 'gulp-cache'});` to something like `var fileCache = new Cache({cacheDirName: '<prefix>-cache'});`. Now gulp-cache will be able to write to `/tmp/<prefix>-cache/default`, and the error should resolve.
 
-### Dashboard deployment
+:::
+
+::::
+
+### Dashboard Deployment
+
+#### Issues
+
+::::note Earthdata Login Error
+
+The dashboard sends you to an Earthdata Login page that has an error reading **"Invalid request, please verify the client status or redirect_uri before resubmitting"**.
+
+:::tip Check your variables and values
+
+Check to see if you are missing or have forgotten to update one or more of your EARTHDATA_CLIENT_ID, EARTHDATA_CLIENT_PASSWORD environment variables (from your app/.env file) and re-deploy Cumulus, or you haven't placed the correct values in them, or you've forgotten to add both the "redirect" and "token" URL to the Earthdata Application.
```

Comment:
```suggestion
Check to see if you are missing or have forgotten to update one or more of your "EARTHDATA_CLIENT_ID", "EARTHDATA_CLIENT_PASSWORD" environment variables (from your app/.env file) and re-deploy Cumulus, or you haven't placed the correct values in them, or you've forgotten to add both the "redirect" and "token" URL to the Earthdata Application.
```

Again, I'm not sure what the standard is for differentiating names so take this suggestion with a grain of salt 😅

Reply:
@botanical Since they are environment variables would they be code references so would it be ` instead of "?

Reply:
Yeah, good point. I think ` would make sense